Modules/clinic/_tkinter.c.h

```diff
@@ -645,6 +652,61 @@ _tkinter_tkapp_willdispatch(TkappObject *self, PyObject *Py_UNUSED(ignored))
     return _tkinter_tkapp_willdispatch_impl(self);
 }
 
+PyDoc_STRVAR(_tkinter_tkapp_dispatching__doc__,
+"dispatching($self, /)\n"
+"--\n"
+"\n"
+"Returns the internal dispatching state\n"
+"\n"
+"Returns 1 if the mainloop is running, or 0 if the mainloop is not running.");
+
+#define _TKINTER_TKAPP_DISPATCHING_METHODDEF    \
+    {"dispatching", (PyCFunction)_tkinter_tkapp_dispatching, METH_NOARGS, _tkinter_tkapp_dispatching__doc__},
+
+static PyObject *
+_tkinter_tkapp_dispatching_impl(TkappObject *self);
+
+static PyObject *
+_tkinter_tkapp_dispatching(TkappObject *self, PyObject *Py_UNUSED(ignored))
+{
+    return _tkinter_tkapp_dispatching_impl(self);
+}
+
+PyDoc_STRVAR(_tkinter_tkapp_setmainloopwaitattempts__doc__,
+"setmainloopwaitattempts($self, new_val, /)\n"
+"--\n"
+"\n"
+"Set number of 100 millisecond mainloop wait attempts.\n"
+"\n"
+"These are used for that call(), var_invoke(), and createcommand().\n"
+"\n"
+"Current default is 10 for a 1 second wait, but future behavior\n"
+"will be equivalent 0.\n"
```

Comment:
```suggestion
"will be equivalent to 0.\n"
```
Sorry for being so picky!